WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -18,7 +18,7 @@ public static class RPCClientExtensions
 		/// <summary>
 		/// Waits for a specific new block and returns useful info about it.
 		/// </summary>
-		/// <param name="timeout">(int, optional, default=0) Time in milliseconds to wait for a response. 0 indicates no timeout.</param>
+		/// <param name="timeout">Time in milliseconds to wait for a response. (int, optional, default=0). 0 indicates no timeout.</param>
```

Comment:
```suggestion
		/// <param name="timeout">Time in milliseconds to wait for a response. 0 indicates no timeout.</param>
```

What I removed here does not provide any information that the IDEA doesn't provide it anyway.